design-documents/graph-ql/coverage/eav/EavGraphQl.graphqls

```diff
@@ -0,0 +1,42 @@
+# Copyright © Magento, Inc. All rights reserved.
+# See COPYING.txt for license details.
+
+type Query {
+    customAttributeMetadata(attributes: [AttributeInput!]!): CustomAttributeMetadata @resolver(class: "Magento\\EavGraphQl\\Model\\Resolver\\CustomAttributeMetadata") @doc(description: "The customAttributeMetadata query returns the attribute type, given an attribute code and entity type") @cache(cacheable: false)
+}
+
+type CustomAttributeMetadata @doc(description: "CustomAttributeMetadata defines an array of attribute_codes and entity_types") {
+    items: [Attribute] @doc(description: "An array of attributes")
+}
+
+type Attribute @doc(description: "Attribute contains the attribute_type of the specified attribute_code and entity_type") {
+    attribute_code: String @doc(description: "The unique identifier for an attribute code. This value should be in lowercase letters without spaces.")
+    entity_type: String @doc(description: "The type of entity that defines the attribute")
+    attribute_type: String @doc(description: "The data type of the attribute")
+    input_type: String @doc(description: "The frontend input type of the attribute")
+    attribute_options: [AttributeOption] @resolver(class: "Magento\\EavGraphQl\\Model\\Resolver\\AttributeOptions") @doc(description: "Attribute options list.")
+    storefront_properties: StorefrontProperties
+}
+
+type StorefrontProperties {
+   used_in_product_listing: Boolean
+   position: Int
+   visible_on_catalog_storefront: Boolean
+   use_in_layered_navigation: UseInLayeredNavigationOptions
```

Comment:
Perhaps name these `use_in_layered_navigation`, `use_in_product_listing`, and `use_in_catalog` for consistency?

Reply:
I understand these names come directly from the backend.

Reply:
Yes, the idea was to reflect the same configuration names as in the backend. So there will be consistency in dev docs too. @keharper can help with the naming.